### ISSUE TITLE
About COCO dataset

### ISSUE THREAD

**Issue by xuehaouwa** [yaoqi-zd/SGAN#10]
Thank you for your repo.
I noticed you reported results on the COCO dataset. Do you have scripts for processing COCO for both classification and segmentation?

**Comment by yaoqi-zd** [comment]
sorry, my school account has been expired now and I cannot get access to the lab machine, the COCO-related codes are messy and need to be refactored before released.

The biggest difference between COCO and VOC-related codes is the loss to train classification network. For coco dataset, since most of the classes are absent in each image, we use a simple balanced sigmoid cross entropy loss to amplify loss of classes that are present in the training images.

I also remember I've adjusted the \alpha value(set higher) when generating the seeds.

hope the above information can help you.

**Comment by xuehaouwa** [comment]
Thank you very much!
